pkgs/applications/graphics/fontmatrix/default.nix

```
@@ -1,23 +1,29 @@
-{ stdenv, fetchFromGitHub, cmake, qt4 }:
+{ lib, mkDerivation, fetchpatch, fetchFromGitHub, cmake, qttools, qtwebkit }:
 
-stdenv.mkDerivation rec {
+mkDerivation rec {
   pname = "fontmatrix";
-  version = "0.6.0";
+  version = "0.6.0-qt5";
 
   src = fetchFromGitHub {
-    owner = "fontmatrix";
+    owner = "fcoiffie";
     repo = "fontmatrix";
-    rev = "v${version}";
-    sha256 = "0aqndj1jhm6hjpwmj1qm92z2ljh7w78a5ff5ag47qywqha1ngn05";
+    rev = "1ff8382d8c85c18d9962918f461341ff4fe21993";
+    sha256 = "0yx1gbsjj9ddq1kiqplif1w5x5saw250zbmhmd4phqmaqzr60w0h";
   };
 
-  buildInputs = [ qt4 ];
+  # Add missing QAction include
+  patches = [ (fetchpatch {
+    url = "https://github.com/fcoiffie/fontmatrix/commit/dc6de8c414ae21516b72daead79c8db88309b102.patch";
+    sha256 = "092860fdyf5gq67jqfxnlgwzjgpizi6j0njjv3m62aiznrhig7c8";
+  })];
+
+  buildInputs = [ qttools qtwebkit ];
 
   nativeBuildInputs = [ cmake ];
 
   hardeningDisable = [ "format" ];
 
-  meta = with stdenv.lib; {
+  meta = with lib; {
     description = "Fontmatrix is a free/libre font explorer for Linux, Windows and Mac";
     homepage = "https://github.com/fontmatrix/fontmatrix";
     license = licenses.gpl2;
```

```suggestion
    license = licenses.gpl2Plus;
```
source: https://github.com/fontmatrix/fontmatrix/blob/master/src/activationwidget.cpp#L8